tests/pytorch/test_pytorch_model_export.py

```diff
@@ -725,6 +729,23 @@ def test_load_pyfunc_succeeds_when_data_is_model_file_instead_of_directory(
     )
 
 
+@pytest.mark.large
+@pytest.mark.parametrize("scripted_model", [True, False])
+def test_save_state_dict(sequential_model, model_path, data):
+
+    mlflow.pytorch.save_state_dict(state_dict=sequential_model.state_dict(), path=model_path)
+
+    model = get_sequential_model()
+    model_state_dict = mlflow.pytorch.load_state_dict(model_path)
+    model.load_state_dict(model_state_dict)
+    model.eval()
+    assert state_dict_equal(model_state_dict, sequential_model.state_dict())
```

Comment:
```suggestion
    state_dict = sequential_model.state_dict()
    mlflow.pytorch.save_state_dict(state_dict=state_dict, path=model_path)

    model = get_sequential_model()
    loaded_state_dict = mlflow.pytorch.load_state_dict(model_path)
    model.load_state_dict(loaded_state_dict)
    model.eval()
    assert state_dict_equal(loaded_state_dict, state_dict)
```

Reply:
I guess it makes sense to keep `get_sequential_model` before loading the state dict. First we define the model and then load it.